test/opexecuter.cpp

```diff
@@ -312,6 +317,18 @@ void OpExecuter::ExecuteOnNGraph() {
   // ng function should get same number of outputs
   ASSERT_EQ(expected_output_datatypes_.size(), ng_function->get_output_size());
 
+  // For debug
+  // Serialize to nGraph if needed
+  if (std::getenv("NGRAPH_ENABLE_SERIALIZE") != nullptr) {
+    std::string file_name = "unit_test_" + test_op_type_ + ".json";
+    NGRAPH_VLOG(0) << "Serializing graph to: " << file_name;
+    std::string js = ngraph::serialize(ng_function, 4);
+    {
```

Comment:
Why do you need this extra scope? Also - you may consider a try catch here i.e., to catch potential exceptions thrown by the ofstream <<. It's a good practice to catch exceptions especially during I/O.

Reply:
In which case, lets make similar changes here too:
https://github.com/NervanaSystems/ngraph-tf/blob/55f9f7ee133a94e05f41fedc4518b81a4c50563f/src/ngraph_encapsulate_op.cc#L277

Reply:
Agree with @sayantan-nervana - please add this change in this PR.